tests/store/api/test_store_api_status.py

```diff
@@ -120,6 +120,82 @@ def test_case_not_in_observations_to_upload(sample_store):
     assert analysis.family not in observations_to_upload
 
 
+def test_analyses_to_upload_when_not_completed_at(helpers, sample_store):
+    """Test analyses to upload with no completed_at date"""
+    # GIVEN a case with completed analysis
+    helpers.add_analysis(store=sample_store)
+
+    # WHEN no analyses are completed
+    records = [analysis_obj for analysis_obj in sample_store.analyses_to_upload()]
+
+    # THEN no analysis object should be returned
+    assert len(records) == 0
+
+
+def test_analyses_to_upload_when_no_pipeline(helpers, sample_store, timestamp):
+    """Test analyses to upload with no pipeline specified"""
+    # GIVEN a case with completed analysis
+    helpers.add_analysis(store=sample_store, completed_at=timestamp)
+
+    # WHEN uploading without a pipeline specified
```

Comment:
This WHEN should be specific for this test 

ditto for the following tests with this text